petab/core.py

```diff
@@ -73,7 +73,11 @@ def get_parameter_df(parameter_file_name):
     """Read the provided parameter file into a `pandas.Dataframe`"""
 
     parameter_df = pd.read_csv(parameter_file_name, sep='\t')
-    parameter_df.set_index(['parameterId'])
+
+    try:
+        parameter_df.set_index(['parameterId'], inplace=True)
+    except KeyError:
+        raise KeyError('Parameter table missing mandatory field `parameterId`.')
```

Comment:
maybe we can for all dataframes define a list of required fields, like
```
parameter_df_req_fields = ['parameterId', 'parameterScale',...]
measurement_df_... = ...
```
That would make testing more straightforward, and the labels could be in one place.

Reply:
Yes, we should and will. Here i would check just for this specific field though. Don't check for more than what is really required.